### ISSUE TITLE
Issue running artisan setup command

### ISSUE THREAD

**Issue by Grusglo** [aimeos/aimeos-laravel#3]
Hello!

Just installed aimeos for laravel but the setup command fails. I have tried both on my production machine and on my local one, but they fail with the same error. Here is the error:

```
  [UnexpectedValueException]                                                                      
  DirectoryIterator::__construct(/home/forge/ext): failed to open dir: No such file or directory  
                                                                                                  


Script php artisan aimeos:setup --option=setup/default/demo:1 handling the post-install-cmd event returned with an error


                      
  [RuntimeException]  
  Error Output:   

```

Thanks in advance

**Comment by aimeos** [comment]
```composer update``` should perform these steps under normal conditions:
* place the aimeos/aimeos-laravel package into the vendor directory
* download the dependent extensions and store them in a newly created ./ext directory in your Laravel application base directory

I guess ```/home/forge/``` is not your Laravel application directory. Does an ```./ext/``` directory exist in your application base directory at all? Can you paste your composer.json file?

**Comment by aimeos** [comment]
Perhaps you forgot to add these lines to your ```composer.json``` file?

```
    "repositories": [ {
        "type": "vcs",
        "url": "https://github.com/aimeos/arcavias-core"
    } ],
    "prefer-stable": true,
    "minimum-stability": "dev",
```

That yields a similar error in my tests

**Comment by Grusglo** [comment]
Hello again,

Thank you for the prompt response. You're right - I did forget to add those lines. I have reinstalled and that error is gone now, however I get another one:

```
[MW_DB_Exception]                                                                  
  SQLSTATE[HY000] [1045] Access denied for user ''@'localhost' (using password: NO)  
                                                                                     


Script php artisan aimeos:setup --option=setup/default/demo:1 handling the post-install-cmd event returned with an error


                      
  [RuntimeException]  
  Error Output:       
```

Thank you again

**Comment by aimeos** [comment]
Is the `config/shop.php` file available? If not, please execute `php artisan vendor:publish`. This should normally be done automatically if you have it in your post-install-cmd/post-update-cmd section of your composer.json file:
```
        "post-update-cmd": [
            "php artisan vendor:publish",
            "php artisan migrate",
            "php artisan aimeos:setup --option=setup/default/demo:1",
            "php artisan aimeos:cache",
            ...
        ]
```

**Comment by aimeos** [comment]
Did this solve your problem?

**Comment by Grusglo** [comment]
Hello

The file did exist, but it still couldn't seem to retrieve the correct information from the env variables. I changed the database values in config/shop to directly retrieve the env variables instead of pulling them through the laravel config (Which I assume was the case before).

I then had an error where the setup could not find the 'name' column in the 'user' table. I solved that by simply adding it myself.


Thank you for the support and a great package.
